### PR TITLE
Add option to disable Mercator calculation for OpenStreetMaps

### PR DESCRIPTION
**Pull request by misimpso**
# Pull Request

## Description
* Add option to disable mercator calculation for the OpenStreetMaps plot.
  * This is needed for plotting data that has already been projected to the Mercator coordinate system.
* Update shapefile advanced example.
* Set plot filename regardless if `save_dir` constructor param has been set.

## Type of Change

Check the type of change this PR addresses:

- [x] Bug
- [x] Feature
- [x] Documentation

## Backwards Compatibility

- [ ] This change breaks backwards compatibility
- [ ] I have bumped the major version

## Documentation

- [X] This change requires a documentation update

## Tests

- [x] This change requires new tests

Add a screenshot or paste your local test results below:
image

## Misc

- [x] I have followed the guidelines for contribution
- [x] I have ensured there aren't other open Pull Requests for the same update/change

## Thanks

Thank you for your interest and taking the time to contribute!

### CODE REVIEW

File: src/wktplot/plots/osm.py [hunk 1]
```diff
@@ -2,18 +2,32 @@
...
+                See this guide for available style attributes:
+                https://docs.bokeh.org/en/2.4.3/docs/reference/plotting/figure.html
+
+        Raises:
+            ValueError: If value for `title` is not a string or None.
+            OSError: If value for `save_dir` is not a directory.
+        """
+
+        super().__init__(*args, **kwargs)
+        self.mapper = StandardMapper if disable_mercator else OpenStreetMapper
```
Comment (misimpso) [inline src/wktplot/plots/osm.py:30]: Should we rename these mapper classes to reference their projection? `StandardMapper` -> `GeometricMapper` and `OpenStreetMapper` -> `MercatorMapper`.

Comment (misimpso) [inline src/wktplot/plots/osm.py:30]: This is the addition to disable the mercator calculation.

**Review by gsornsen — APPROVED** [review]

File: src/wktplot/plots/osm.py (hunk 1 above)
Comment (gsornsen) [reply src/wktplot/plots/osm.py:30]: Yes, I'll certainly find that helpful in the future when I forget which is "Standard"

Comment (misimpso) [reply src/wktplot/plots/osm.py:30]: Ok, will do in a followup PR.

**Pull request merged**